code/__DEFINES/botany.dm

```diff
@@ -0,0 +1,2 @@
+/// A list of possible egg laying descriptions
+#define EGG_LAYING_MESSAGES list("lays an egg.","squats down and croons.","begins making a huge racket.","begins clucking raucously.")
```

Comment:
This is the single most specific file I have ever seen.

Reply:
Yeah it is, I guess chickens get special treatment